cp3pt0-deployment/setup_singleton.sh

```diff
@@ -162,12 +182,15 @@ function print_usage() {
     echo "   --oc string                                    Optional. File path to oc CLI. Default uses oc in your PATH"
     echo "   --yq string                                    Optional. File path to yq CLI. Default uses yq in your PATH"
     echo "   --operator-namespace string                    Optional. Namespace to migrate Cloud Pak 2 Foundational services"
-    echo "   --enable-licensing                             Optional. Set this flag to install ibm-licensing-operator"
+    echo "   -ls, --enable-licensing                        Optional. Set this flag to install ibm-licensing-operator"
+    echo "   -lsNs, --licensing-namespace string            Optional. Set custom namespace for ibm-licensing-operator. Default is ibm-licensing"
```

Comment:
We are breaking the interface here: we change the parameter `-licensingNs` into `--licensing-namespace` which sounds dangerous

Reply:
reverted to -licensingNs

Reply:
fair point